### PR TITLE
Translate creating-a-starter.md file

### PR DESCRIPTION
**Pull request by JuanJavier1979**
Translation of `docs/docs/creating-a-starter.md` file to Spanish

### CODE REVIEW

File: docs/docs/creating-a-starter.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
 ---
-title: Creating a Starter
+title: Creando un _Starter_
 ---
 
-[Starters](/docs/starters/) are boilerplate projects that Gatsby developers can use to set up a new site quickly. Before creating a starter, it may be helpful to peruse the [Gatsby Starter Library](/starters/) to see what already exists and determine how your starter will provide value.
+[_Starters_](/docs/starters/) son proyectos tipo plantilla que los desarrolladores Gatsby pueden usar para configurar un nuevo sitio rápidamente. Antes de crear un _starter_, puede ser útil examinar detenidamente la [Librería de _Starters_ de Gatsby](/starters/) para ver lo que ya existe y determinar cómo tu _starter_ puede aportar valor.
```
Comment (Darking360) [inline docs/docs/creating-a-starter.md]:
```suggestion
[_Los starters_](/docs/starters/) son proyectos tipo plantilla que los desarrolladores Gatsby pueden usar para configurar un nuevo sitio rápidamente. Antes de crear un _starter_, puede ser útil examinar detenidamente la [Librería de _Starters_ de Gatsby](/starters/) para ver lo que ya existe y determinar cómo tu _starter_ puede aportar valor.
```

File: docs/docs/creating-a-starter.md [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
 
-## Basic requirements
+## Requerimientos básicos
 
-For a starter to work properly, it needs to include some files (see the [Hello World starter](https://github.com/gatsbyjs/gatsby-starter-hello-world/) for a barebones example):
+Para que un _starter_ funcione adecuadamente, necesita incluir algunos ficheros (mira el [_starter_ Hola Mundo](https://github.com/gatsbyjs/gatsby-starter-hello-world/) para ver un ejemplo tipo esqueleto):
```
Comment (Darking360) [inline docs/docs/creating-a-starter.md]:
```suggestion
Para que un _starter_ funcione adecuadamente, necesita incluir algunos ficheros (mira el [_starter_ Hola Mundo](https://github.com/gatsbyjs/gatsby-starter-hello-world/) para ver un ejemplo del tipo esqueleto):
```

File: docs/docs/creating-a-starter.md [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
-Although the official starters live in the Gatsby repo, community members can offer their own starters from their own repos. Here's an example of installing a community starter:
+Aunque los _starters_ oficiales están en el repositorio de Gatsby, miembros de la comunidad pueden ofrecer sus propios _starters_ desde sus propios repositorios. Aquí hay un ejemplo de instalación de un _starter_ de la comunidad:
 
-`gatsby new my-app https://github.com/netlify-templates/gatsby-starter-netlify-cms`
+`gatsby new mi-aplicacion https://github.com/netlify-templates/gatsby-starter-netlify-cms`
 
 ## Configurable
 
-Starters should utilize metadata in `gatsby-config.js` wherever possible, as this is typically the first place users will look for site configuration information. Some examples of things you could make configurable in `gatsby-config` include:
+Los _starters_ deben utilizar metadatos en `gatsby-config.js` cuando sea posible, ya que este suele ser el primer sitio donde los usuarios buscarán información para configurar el sitio. Algunos ejemplos de cosas que podrías hacer confirables en el `gatsby-config` son:
```
Comment (Darking360) [inline docs/docs/creating-a-starter.md]:
```suggestion
Los _starters_ deben utilizar metadatos en `gatsby-config.js` cuando sea posible, ya que este suele ser el primer sitio donde los usuarios buscarán información para configurar el sitio. Algunos ejemplos de cosas que podrías hacer configurables en el `gatsby-config` son:
```

**Comment by JuanJavier1979** [comment]
Thank you @Darking360 
Suggestions added :-)

**Pull request merged**